src/sources/webgl.ts

```diff
@@ -24,6 +24,7 @@ type WebGlExtensionsPayload = {
   shaderPrecisions: string[] // ['FRAGMENT_SHADER.LOW_FLOAT=127,127,23...
   extensions: string[] | null // ['ANGLE_instanced_arrays', 'EXT_blend_minmax', 'EXT_color...
   extensionParameters: string[] // ['COMPRESSED_RGB_S3TC_DXT1_EXT(33776)', 'COMPR...
+  supported: boolean[]
```

Comment:
- The name should have more context. For example `supportedExtensions`.
- The fact that this field references to the `extensions` field is not obvious, so this should be mentioned in a comment
- Because the array is all-true in almost all cases, it makes sense to keep only unsupported extensions. It will reduce the payload size. This can be either extension names, or the `extensions`'s indices (with a comment telling about the reference)